tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -572,8 +572,12 @@ def sync_volumes(volumes, index, vol_options, sync_ttl):
             hostname = volumes[
                 'volume%s.brick%s.hostname' % (index, b_index)
             ]
-            if socket.gethostbyname(NS.node_context.fqdn) != \
-                    socket.gethostbyname(hostname):
+            ip = socket.gethostbyname(hostname)
+            node_id = etcd_utils.read("indexes/ip/%s" % ip).value
```

Comment:
So effectively `/indexes/ip` is now going to have entries for all the IPs of machines from different networks. right?

Reply:
yes, we already storing all the ip of particular node with node id in indexes